packages/@sanity/desk-tool/src/panes/document/statusBar/dialogs/ConfirmDialog.tsx

```diff
@@ -19,15 +20,17 @@ export function ConfirmDialog(props: {
   const {dialog, referenceElement} = props
 
   return (
```

Comment:
Maybe add a commenting explaining the theme color provider usage?

```suggestion
  // Wrapping in ThemeColorProvider to avoid the ConfirmDialog inheriting tone of the footer
  return (
```